src/AddSkillForm.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+import "./AddSkillForm.css";
+
+function AddSkillForm({ onSubmit }) {
+  const [name, setName] = useState("");
+  const [proficiency, setProficiency] = useState("");
+  const [logo, setLogo] = useState("");
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    const skillData = {
+      name,
+      proficiency,
+      logo,
+    };
+
+    try {
+      const response = await fetch("/resume/skill", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(skillData),
+      });
+
+      const result = await response.json();
+
+      if (response.ok) {
+        console.log(`Skill added with ID: ${result.id}`);
```

Comment:
nit: we don't need to show the id to the user here

Reply:
@Pradyuman7 Fixed this problem.